setup.py

```diff
@@ -92,7 +92,7 @@
     "anthropic": ["anthropic>=0.23.1"],
     "mistral": ["mistralai>=0.2.0"],
     "groq": ["groq>=0.9.0"],
-    "ollama": ["ollama>=0.2.1"],
+    "ollama": ["ollama>=0.2.1", "fix_busted_json"],
```

Comment:
does the  "fix-busted-json" need to be on a separate line?  It didn't install automatically for me.

Reply:
Had the same issue myself and got the exception `ImportError: Please install ollama to use the Ollama API`.
Also resolved by installing fix_busted_json.

Note: As I am experimenting through the devcontainer docker image provided by autogen, so to get it working I needed to modify
https://github.com/microsoft/autogen/blob/main/.devcontainer/dev/Dockerfile#L51 to
`RUN pip install numpy pandas matplotlib seaborn scikit-learn requests urllib3 nltk pillow pytest beautifulsoup4 ollama fix-busted-json`

Perhaps these files Docker files should be updated as part of this PR?

- https://github.com/microsoft/autogen/blob/main/.devcontainer/dev/Dockerfile
- https://github.com/microsoft/autogen/blob/main/.devcontainer/full/Dockerfile

Reply:
Thanks for working through this. The docker files will contain just the generic libraries for AutoGen. To run Ollama, or any other of the specific client classes, we'll need to manually pip install those packages within the DevContainer. For a user, they'll use the `pip install pyautogen[ollama]` and that will install the ollama and fix_busted_json packages.